async-nats/tests/client_tests.rs

```diff
@@ -266,15 +268,21 @@ mod client {
             nats_server::run_basic_server(),
         ];
 
-        let mut client = async_nats::connect(
-            servers
-                .iter()
-                .map(|server| server.client_url().parse::<ServerAddr>().unwrap())
-                .collect::<Vec<ServerAddr>>()
-                .as_slice(),
-        )
-        .await
-        .unwrap();
+        let mut client = async_nats::ConnectOptions::new()
```

Comment:
Maybe just leave this test as it was, with the default behavior?

```suggestion
        let mut client = async_nats::ConnectOptions::new()
```